composer.json

```diff
@@ -11,6 +11,9 @@
     ],
     "require": {
         "php": ">=7.4",
+        "ext-bcmath": "*",
+        "ext-gmp": "*",
+        "ext-json": "*",
```

Comment:
Does JSON need adding here since it wasn't in the original documentation?

Reply:
Ah, ok, yeah. In fact, I just removed PHP7 from the test matrix [here](https://github.com/Flagsmith/flagsmith-php-client/pull/80), so I think we can probably remove JSON from this list.

Reply:
Fair, sorry about that.